cpp/oneapi/dal/algo/knn/backend/gpu/infer_kernel_impl_dpc_distr.hpp

```diff
@@ -371,16 +365,10 @@ class knn_callback_distr {
 
         const auto& [first, last] = bnds;
         ONEDAL_ASSERT(last > first);
-        auto& queue = this->queue_;
-
-        bk::event_vector ndeps{ deps.cbegin(), deps.cend() };
-        auto sq_event = copy_with_sqrt(queue, inp_dts, inp_dts, deps);
```

Comment:
Can you clarify please how you avoid the usage of copy_with_sqrt func?

Reply:
It's done here already - https://github.com/oneapi-src/oneDAL/pull/2733/files#diff-74259405865e71161423cc6a67bfeef97953a7b9adc8ec21ef8f0162fa1a9d13L283